src/lib/db/feature-toggle-client-store.ts

```diff
@@ -93,6 +96,7 @@ export default class FeatureToggleClientStore
                 'fs.feature_name',
                 'features.name',
             )
+            .leftJoin('feature_tag as ft', 'ft.feature_name', 'features.name')
```

Comment:
I am unsure if we should do this join for client SDK calls as well, I am considering the unknown implication on performance. Wdyt @chriswk?